src/AnimationControl.tsx

```diff
@@ -94,178 +78,170 @@ function AnimationControl({
-                <Tooltip title="Reset step size">
-                    <Button onClick={() => roundAndSetStepSize(1)}>
-                        <RestartAltIcon />
-                    </Button>
-                </Tooltip>
-            </Stack>
-            <Box display="flex" justifyContent="center">
-                <ButtonGroup size="large" variant="outlined">
-                    <Tooltip title={`Backward one step (${STEP_BACKWARD_KEY})`}>
-                        <Button onClick={onSkipBackward}>
+        <Box sx={{
+            position: 'fixed',
+            bottom: 0,
+            left: 0,
+            right: 0,
+            zIndex: 1000,
+            backgroundColor: 'rgba(18, 18, 18, 0.95)',
+            backdropFilter: 'blur(20px)',
+            borderTop: '1px solid rgba(255, 255, 255, 0.1)',
+            px: 3,
+            py: 1.5,
+        }}>
+            <Stack direction="row" alignItems="center" justifyContent="space-between">
+                <Box sx={{ width: 48 }} />
+
+                <Stack direction="row" spacing={2} alignItems="center">
+                    <Tooltip title="Previous step (←)">
+                        <IconButton
+                            onClick={onSkipBackward}
+                            size="small"
+                            sx={{
+                                '&:hover': {
+                                    backgroundColor: 'rgba(255, 255, 255, 0.1)',
+                                }
+                            }}
+                        >
                             <SkipPreviousIcon />
-                        </Button>
+                        </IconButton>
                     </Tooltip>
-                    <Tooltip title={(playAnimation ? "Pause" : "Play") + ` (spacebar)`}>
-                        <Button onClick={() => onPlayAnimationChange(!playAnimation)}>
-                            {playAnimation ?
-                            <PauseTwoToneIcon /> :
-                            <PlayArrowIcon />}
-                        </Button>
+
+                    <Tooltip title={(playAnimation ? "Pause" : "Play") + " (space)"}>
+                        <IconButton
+                            onClick={() => onPlayAnimationChange(!playAnimation)}
+                            sx={{
+                                width: 48,
+                                height: 48,
+                                backgroundColor: 'primary.main',
+                                '&:hover': {
+                                    backgroundColor: 'primary.dark',
+                                }
+                            }}
+                        >
+                            {playAnimation ? <PauseTwoToneIcon /> : <PlayArrowIcon />}
+                        </IconButton>
                     </Tooltip>
-                    <Tooltip title={`Forward one step (${STEP_FORWARD_KEY})`}>
-                        <Button onClick={onSkipForward}>
+
+                    <Tooltip title="Next step (→)">
```

Comment:
```suggestion
                    <Tooltip title="Next step (🡒)">
```
this character looks a bit nicer